CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+This document outlines major changes between releases.
+
+## [Unreleased]
+
+New features:
+
+Behaviour changes:
+ * add generic Hash/Address parameters to `DBFT` service (#94)
+ * remove custom payloads implementation from default `DBFT` service configuration
+   (#94)
+
+Improvements:
+ * untie `dbft` module from `github.com/nspcc-dev/neo-go` dependency (#94)
+
+Bugs fixed:
+
+## 0.1.0 (13 Feb 2024)
```

Comment:
You can safely put 15 May 2023 here, the date of commit.

Reply:
Fixed.